.github/workflows/pr-test.yml

```
@@ -0,0 +1,79 @@
+      fail-fast: false  # 첫 번째 실패가 발생해도 나머지 작업들은 계속 진행됨
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.ACTION_TOKEN }}
+          submodules: true
+          fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
+      - name: Set up JDK 17
+        uses: actions/setup-java@v4
+        with:
+          java-version: 17
+          distribution: 'zulu' # Alternative distribution options are available
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)